api/files/serializers.py

```diff
@@ -423,6 +432,13 @@ def get_download_link(self, obj):
             view_only=self.context['request'].query_params.get('view_only'),
         )
 
+    def get_render_link(self, obj):
+        file = self.context['file']
+        mfr_url = get_mfr_url(file.target, file.provider)
+        download_url = self.get_download_link(obj)
```

Comment:
I spent awhile trying to optimize because the download link is now being built multiple times - either caching the download URL on the `serializer_context` or caching the file's guid if it existed.  If the value wasn't cached, I built it here (in the case of FileVersionsList for example, where there would be many URL's, so it didn't change many of the endpoints.  

These changes actually seemed to slow things down (I suspect because I was making extra requests to fetch the file to then build the url/fetch the guid), so I reverted.